apps/address-service/domains/common/utils/healthchecks.js

```diff
@@ -0,0 +1,50 @@
+const { get } = require('lodash')
+
+const conf = require('@open-condo/config')
+
+const { DADATA_PROVIDER } = require('@address-service/domains/common/constants/providers')
+const { DadataSuggestionProvider } = require('@address-service/domains/common/utils/services/suggest/providers/DadataSuggestionProvider')
+
+const getAddressProviderBalanceHealthCheck = () => {
+    return {
+        name: 'address-provider-balance',
+        prepare: () => {
+            const provider = get(conf, 'PROVIDER')
+
+            if (provider === DADATA_PROVIDER) {
+                this.addressProvider = new DadataSuggestionProvider()
```

Comment:
The provider uses `req.id` for logging. You need to pass args to DadataSuggestionProvider:
```js
DadataSuggestionProvider({ req: { id: 'some id' } })
```
Please, see the constructor.

Reply:
I should create req.id by myself? It should be something like uuid? @AleX83Xpert

Reply:
Yes, UUID will fit.